website_form_recaptcha/data/ir_config_parameter_data.xml

```diff
@@ -6,18 +6,13 @@
 -->
 
 
-<openerp>
-    <data noupdate="1">
-        <record id="recaptcha_key_site" model="ir.config_parameter">
-            <field name="display_name">ReCaptcha Site Key</field>
-			<field name="key">recaptcha.key.site</field>
-			<field name="value">Obtain from Google</field>
-        </record>
-		<record id="recaptcha_key_secret" model="ir.config_parameter">
-            <field name="display_name">ReCaptcha Secret Key</field>
-			<field name="key">recaptcha.key.secret</field>
-			<field name="value">Obtain from Google</field>
-			<field name="group_ids" eval="[(4, ref('base.group_system'))]" />
-        </record>
-    </data>
-</openerp>
+<odoo noupdate="1">
+    <record id="recaptcha_key_site" model="ir.config_parameter">
+		<field name="key">recaptcha.key.site</field>
+		<field name="value">Obtain from Google</field>
+    </record>
+	<record id="recaptcha_key_secret" model="ir.config_parameter">
+		<field name="key">recaptcha.key.secret</field>
+		<field name="value">Obtain from Google</field>
```

Comment:
Please use coherent indentation with 4 spaces 😄

Reply:
Pretty sure this was my bad. Old IDE really loved tabs